tests/compat_ip.list

```diff
@@ -1,6 +1,6 @@
 198.51.100.200
 198.51.100.1-198.51.100.100
 203.0.113.100/30
```

Comment:
should this be /24?

Reply:
A /30 is "inside" a /24 so it is safe to use.